AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -156,14 +162,21 @@ - (void)setChildren:(NSArray *)children
 
 - (id<ASLayoutable>)child
 {
-  return _child;
+  return [_children firstObject];
```

Comment:
The previous code would not crash if _child was nil; this case may not have been permitted by assertions, but it would succeed at least in production builds.  We should ensure this does too.